andela/mmdp_site
# MMDP -containerize project

#### What does this PR do?
Creates docker images and containers required to run the project

#### Description of Task to be completed?
Run the project in production using docker compose

#### How should this be manually tested?

-  Ensure you have docker machine and docker compose installed
- In the project root folder, run `docker-compose up`
- Should you make changes to the docker file or other configurations, run `docker-compose up --build`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
![screenshot 2019-01-29 at 11 51 35](https://user-images.githubusercontent.com/23254784/51895910-55187200-23bc-11e9-95b3-5654d6a9da8a.png)

#### Screenshots (if appropriate)
#### Questions:
